### PR TITLE
expose version on API, fix travis release

### LINKED ISSUE #30: install-utask.sh is not correctly updated
* **I'm submitting a ...**
  - [ x] bug report

* **Do you want to request a *feature* or report a *bug*?**
Bug report
install-utask.sh provided is not correctly updated when we download it :
Steps to reproduce :
```bash
wget https://github.com/ovh/utask/releases/latest/download/install-utask.sh
docker-compose up
```

* **What is the current behavior?**
```bash
docker-compose up
Starting utask_db_1    ... done
Starting utask_utask_1 ... done
Attaching to utask_utask_1, utask_db_1
utask_1  | time="2019-11-21T08:24:07Z" level=info msg="[DatabaseConfig] Using 50 max open connections, 30 max idle connections, 60 seconds timeout"
utask_1  | time="2019-11-21T08:24:08Z" level=fatal msg="dial tcp 172.18.0.3:5432: connect: connection refused"
db_1     | LOG:  database system was shut down at 2019-11-21 08:24:00 UTC
```

* **What is the expected behavior?**
Application is starting

* **Which version of uTask are you using?**
Latest version - 1.0.1 (see wget command)

* **Please tell us about your environment**
```bash
lsb_release -d
Description:	Ubuntu 18.04.2 LTS
```

* **Other information**
This commit is not correctly applied to install-task.sh provided
https://github.com/ovh/utask/commit/8ecc5dd68c08e0bdb4f93c72c06d9606f930f771

In docker-compose.yaml : 
```yaml
    depends_on:
      - db
```
is missing in utask section

After adding this part, application is starting correctly

--
I am not familiar on how install-script.sh is generated for the release, but if you have few guidelines, I will be glad to open a PR

### PR DESCRIPTION
**Pull request by nomonamo**
Signed-off-by: Pablo Pérez Schröder <2639770+nomonamo@users.noreply.github.com>

should fix #30 

- travis + goreleaser is unable to create a draft release upon finding modified files (go.mod + go.sum)

- attempting to upload generated install script on tagged release

- expose version and commit hash on API

### CODE REVIEW

File: api/server.go [hunk 1]
```diff
@@ -241,12 +241,21 @@ func (s *Server) build(ctx context.Context) {
...
 				maintenanceMode,
 				tonic.Handler(handler.CancelResolution, 204))
+
 			//	authRoutes.POST("/resolution/:id/rollback",
 			//		[]fizz.OperationOption{
 			// 			fizz.Summary(""),
 			//		},
 			//		tonic.Handler(handler.ResolutionRollback, 200))
 
+			authRoutes.GET("/",
```
Comment (w3st3ry) [inline api/server.go]: Thanks! Forget to add that in the last PR.

File: .travis.yml [hunk 2]
```diff
@@ -28,9 +28,23 @@ before_script:
...
   - make test-travis
+  - make
+  - make release
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_TOKEN
+  file: 
+    - install-utask.sh
+    - utask
```
Comment (w3st3ry) [inline .travis.yml]: Why are you uploading the binary?
Goreleaser already do that.

File: .travis.yml [hunk 3]
```diff
@@ -28,9 +28,23 @@ before_script:
 
 script:
   - make test-travis
+  - make
+  - make release
+
+deploy:
```
Comment (w3st3ry) [inline .travis.yml]: It wont erase the release made by Goreleaser?

File: .travis.yml (hunk 2 above)
Comment (nomonamo) [reply .travis.yml]: my bad, wasn't aware it was redundant

File: .travis.yml (hunk 3 above)
Comment (nomonamo) [reply .travis.yml]: I assumed it would all be aggregated under the same release, but I should check first

Comment (w3st3ry) [reply .travis.yml]: Maybe checkout https://goreleaser.com/ci/
This could simplify the `Makefile` within goreleaser step.

File: .travis.yml [hunk 4]
```diff
@@ -28,9 +28,12 @@ before_script:
 
 script:
   - make test-travis
+  - make
```
Comment (w3st3ry) [inline .travis.yml]: Still useless now.

Comment (nomonamo) [reply .travis.yml]: fair enough

**Pull request merged**